pkg/client/src/index.tsx

```diff
@@ -12,7 +12,11 @@ import { NinjaErrorBoundary } from "@app/ninja-error-boundary";
 import { LocalStorageContextProvider } from "@app/context/LocalStorageContext";
 import { KeycloakProvider } from "@app/common/KeycloakProvider";
 
-i18n.init();
+i18n.init({
+  react: {
+    useSuspense: false,
+  },
+});
```

Comment:
Is this required for this PR, as being a blocker ? 
Because if not I'd like to keep it separate for impact reasons (too close to code freeze).

Reply:
Yeah it is required for this PR.

Reply:
NVM - Was able to remove it in favor of using suspense to wrap children in the KeycloakProvider.